playbooks/rhv/provisioning-vars.yaml.example

```diff
@@ -0,0 +1,82 @@
+---
+###########################
+# Engine Connection
+###########################
+compatibility_version: 4.2
+engine_url:       # https://engine.example.com/ovirt-engine/api
+engine_user:      # admin@internal
+engine_password:  # secret
+
+# CA file copied from engine:/etc/pki/ovirt-engine/ca.pem
+# path is relative to openshift-cluster directory
+engine_cafile:    # ../ca.pem
+
+data_center_name: # Default
+openshift_rhv_cluster:      # Default
+openshift_rhv_data_store: # vmstore
+openshift_rhv_ssh_key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
+
+##########################
+# Template Creation
+# https://github.com/oVirt/ovirt-ansible-image-template
+##########################
+qcow_url: # https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2c
+image_path: "{{ lookup('env', 'HOME') }}/Downloads/{{ template_name }}.qcow2"
+template_name: # rhel75
+template_cluster: "{{ openshift_rhv_cluster }}"
+template_memory: 8GiB
+template_cpu: 1
+template_disk_storage: "{{ openshift_rhv_data_store }}"
+template_disk_size: 60GiB
+template_nics:
+  - name: nic1
+    profile_name: # ovirtmgmt
+    interface: virtio
+
+##########################
+# Virtual Machines
+##########################
+wait_for_ip: true # Required to gather IP addresses from amchines after
+vm_infra_wait_for_ip_retries: 10 # Higher than default to wait for all IPs on all VMs
+vm_infra_wait_for_ip_delay: 8
+
+# Virtual machine profile for master nodes
+# Differs from node profile by 16GiB RAM, extra disk for etcd
+master_vm:
+  cluster: "{{ openshift_rhv_cluster }}"
+  template: "{{ template_name }}"
+  memory: 16GiB
+  cores: 2
+  high_availability: true
+  disks:
+    - size: 15GiB
```

Comment:
How are the disks mounted to the vm?
To what path?

Reply:
They're mounted during prerequisites.yml, in the `container_runtime` role's `extra_storage_setup` task:
https://github.com/openshift/openshift-ansible/blob/master/roles/container_runtime/tasks/extra_storage_setup.yml

Vars set in master and node group vars: https://github.com/openshift/openshift-ansible/pull/9570/files#diff-8aa4dd60378793fbc8c2f195dec32c70R65